Content.Shared/Access/Systems/SharedIdCardSystem.cs

```diff
@@ -138,6 +138,7 @@ public bool TryChangeJobTitle(EntityUid uid, string? jobTitle, IdCardComponent?
         if (id.LocalizedJobTitle == jobTitle)
             return true;
         id.LocalizedJobTitle = jobTitle;
+        id.JobTitle = jobTitle;
```

Comment:
The JobTitle datafield is for a localization id and should not be set to a normal string like this.
```suggestion
```

Reply:
Should be fixed now.